<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHistogramClass</class>
 <widget class="QDialog" name="DlgHistogramClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_16">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditYMax">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLineEdit" name="lineEditXMax">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QLineEdit" name="lineEditXMin">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBox_nBins">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string># of bins:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2" colspan="2">
      <widget class="QLineEdit" name="lineEditYMin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="4" colspan="4">
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_15">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>CF</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QDial" name="dial">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>1023</number>
            </property>
            <property name="value">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_returnCF">
            <property name="text">
             <string>Return CF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_ReDrawGraph_dial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditXMax</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_ReDrawGraph_limits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditXMin</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_ReDrawGraph_limits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditYMax</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_ReDrawGraph_limits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditYMin</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_ReDrawGraph_limits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_nBins</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_DrawGraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_returnCF</sender>
   <signal>released()</signal>
   <receiver>DlgHistogramClass</receiver>
   <slot>SLT_ReturnCF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SLT_DrawGraph()</slot>
  <slot>SLT_ReDrawGraph_dial()</slot>
  <slot>SLT_ReDrawGraph_limits()</slot>
  <slot>SLT_ReturnCF()</slot>
 </slots>
</ui>
